src/hooks/tradingView/useTradingView.ts

```diff
@@ -118,7 +118,7 @@ export const useTradingView = ({
       tvWidgetRef.current = null;
       setIsChartReady(false);
     };
-  }, [selectedLocale, selectedNetwork, !!marketId, hasPriceScaleInfo]);
+  }, [selectedLocale, selectedNetwork, marketId, hasPriceScaleInfo]);
```

Comment:
revert back to `!!marketId` or use `Boolean(marketId)`. That dep was acting as a guard for initializing the chart. After the chart is initialized we want to prevent re-initializing when marketId changes.